src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -9,6 +10,24 @@ public class RegistrationServiceImpl implements RegistrationService {
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throw new RegistrationUserException("User cannot be null");
+        }
+        if (user.getLogin() == null || user.getLogin().length() < 6) {
```

Comment:
Move the size limit to the constant fields (also, use it in the exception message)